less/slider.less

```diff
@@ -137,53 +124,51 @@
 .slider-widget {
     .slider-holder {
         .rangeslider__handle {
-          background-image: none;
-          border: none;
-          background-color: @item-color;
-          &:after {
-            content: none;
-          }
-          &:hover {
-            .no-touch & {
-              background-color: @item-hover-color;
+            background-image: none;
+            border: none;
+            background-color: @item-color;
+            &:after {
+                content: none;
+            }
+            &:hover {
+                .no-touch & {
+                    background-color: @item-hover-color;
+                }
             }
-          }
         }
         .rangeslider__fill {
-          background-color: @item-color;
-          transition: background-color linear 200ms;
+            background-color: @item-color;
+            transition: background-color linear 200ms;
         }
         .rangeslider {
-			background-color: @background-color;
-			.dir-rtl & {
-				transform: rotateY(180deg);
-				-webkit-transform: rotateY(180deg);
-				-ms-transform: rotate(180deg);/*IE9 does not support 3D transform*/
-				-moz-transform: rotateY(180deg);
-				-o-transform: rotateY(180deg);
-			}
+            background-color: @background-color;
+            .dir-rtl & {
+                transform: rotateY(180deg);
+                -webkit-transform: rotateY(180deg);
+                -ms-transform: rotate(180deg);
+                /*IE9 does not support 3D transform*/
+                -moz-transform: rotateY(180deg);
+                -o-transform: rotateY(180deg);
+            }
         }
-
         .rangeslider--disabled {
-          filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=100);
-          opacity: 1;
-          .rangeslider__handle {
-            border: none;
-            filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);
-            opacity: 0;
-          }
+            filter: progid: DXImageTransform.Microsoft.Alpha(Opacity=100);
```

Comment:
please remove spacing at progid:DXImageTransform.Microsoft.Alpha(Opacity=100;
this broke less compilation